Marlin/src/libs/MAX31865.cpp

```diff
@@ -152,22 +145,13 @@ void MAX31865::begin(max31865_numwires_t wires, float zero, float ref) {
   OUT_WRITE(_cs, HIGH);
 
   if (_sclk != TERN(LARGE_PINMAP, -1UL, -1)) {
-    // Define pin modes for Software SPI
-    #ifdef MAX31865_DEBUG
-      SERIAL_ECHOLN("Initializing MAX31865 Software SPI");
-    #endif
-
-    OUT_WRITE(_sclk, LOW);
-    SET_OUTPUT(_mosi);
-    SET_INPUT(_miso);
+    softSpiBegin(SPI_QUARTER_SPEED); // Define pin modes for Software SPI
```

Comment:
Should this be `softSpiBegin(TERN(TARGET_LPC1768, SPI_QUARTER_SPEED, TERN(ARDUINO_ARCH_STM32, SPI_CLOCK_DIV4, 500000));` ?

Reply:
`softSpiBegin` currently takes a shift value and uses it in an equation that gives a nanoseconds result. 
```cpp
_spi_delay = (100UL << spi_speed) / 3;
```
Those other values `SPI_CLOCK_DIV4` and `500000` are quite large numbers and seem to relate to the clock cycles speed, which would need to be translated back to nanoseconds in such usage. I have no idea how those values are otherwise used by `SPI.beginTransaction` when passed, whether they are architecture-specific, and so on…
```cpp
SPI.beginTransaction(spiConfig);
```
If you're very interested in the topic and want to be more involved in this PR, then please make a study of those SPI implementations and let us know what's going on under the hood and whether our nanoseconds delays are a good idea.